.gitignore

```diff
@@ -2,6 +2,11 @@
 *.csv
 *.tsv
 *.rar
+*.logs
+.vscode/
 .ipynb_checkpoints/
+catboost_info/
 notebook/__pycache__/
```

Comment:
kalo ngeignore di semua level folder gimana ya?

Reply:
tanpa pake '/' works ga?
https://stackoverflow.com/questions/35916658/how-to-git-ignore-ipython-notebook-checkpoints-anywhere-in-repository